website_sale_checkout_adj/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgp1).
+{
+    'name': 'View Adjustments on Website Sale',
+    'version': '10.0.0.0.0',
+    'category': 'Website',
+    'summary': '',
+    'description': """
+Show a message to prompt inputting the fullname below the name field.
+And make the zip_code field mandatory.
+    """,
```

Comment:
```suggestion
```
Let's remove these lines since the readme takes over this role.

Reply:
Thankyou, I edited it.